plugin/src/sbt-test/sbt-scala3-migrate/type-inference-migration/build.sbt

```diff
@@ -2,6 +2,8 @@ lazy val `type-inference-migration` = project
   .in(file("."))
   .settings(
     scalaVersion := "2.13.5",
+    libraryDependencies +=
+      "ch.epfl.scala" %% "scalafix-rules" % "0.9.26" % Test cross CrossVersion.for3Use2_13, // to create a test configuration
```

Comment:
I don't think this is needed. `Test` is a default config in sbt, it is always created.

Reply:
ah ok!